packages/gui/src/components/plot/PlotHarvester.tsx

```diff
@@ -0,0 +1,158 @@
+import React, { useEffect, useState } from 'react';
+import { Trans } from '@lingui/macro';
+import { useToggle } from 'react-use';
+import { Accordion, Flex, FormatBytes, Tooltip, FormatLargeNumber } from '@chia/core';
+import { useGetHarvesterQuery } from '@chia/api-react';
+import { Badge, Typography, Chip } from '@mui/material';
+import { ExpandMore, ExpandLess } from '@mui/icons-material';
+import { Box, Tab, Tabs } from '@mui/material';
+import PlotHarvesterPlots from './PlotHarvesterPlots';
+import PlotHarvesterPlotsNotFound from './PlotHarvesterPlotsNotFound';
+import PlotHarvesterPlotsFailed from './PlotHarvesterPlotsFailed';
+import PlotHarvesterPlotsDuplicate from './PlotHarvesterPlotsDuplicate';
+import PlotHarvesterState from './PlotHarvesterState';
+
+export type PlotHarvesterProps = {
+  nodeId: string;
+  host: string;
+  port: string;
+  expanded?: boolean;
+};
+
+export default function PlotHarvester(props: PlotHarvesterProps) {
+  const { nodeId, host, expanded: expandedDefault = false } = props;
+
+  const { plots, noKeyFilenames, failedToOpenFilenames, duplicates, totalPlotSize, initialized } = useGetHarvesterQuery({
+    nodeId,
+  });
+
+  const [activeTab, setActiveTab] = useState<'PLOTS' | 'NOT_FOUND' | 'FAILED' | 'DUPLICATE'>('FAILED');
+  const [expanded, toggleExpand] = useToggle(expandedDefault);
+  const simpleNodeId = `${nodeId.substr(0, 6)}...${nodeId.substr(nodeId.length - 6)}`;
+  const isLocal = host === '127.0.0.1';
```

Comment:
Do we need to handle IPv6 localhost as well? `::1`

Reply:
I already added it. Thank you